dockers/actions-plan-preview/planpreview.go

```diff
@@ -257,3 +264,22 @@ func groupApplicationResults(apps []ApplicationResult) (changes, pipelines, quic
 	}
 	return
 }
+
+func getActionsURL() string {
+	serverURL := os.Getenv(githubServerURLEnv)
+	if serverURL == "" {
+		return ""
```

Comment:
not sure about the behavior of the browser when click into a link tag with empty link, but I guess the default/empty value for an url should be `#` instead of empty string "", wdyt

Reply:
Currently, if url is not successfully obtained, actions log badge is not included in comment. So I think this is not cause problem.
If the behavior of not showing the badge itself is strange, maybe `#` is better.

Reply:
Got it, thanks 👍